README.md

```diff
@@ -111,7 +111,10 @@ spec:                  # spec (used to be che-plugin.yaml)
   containers:          # optional; sidecar containers for plugin
     - image:
       name:              # name used for sidecar container
-      memorylimit:       # Kubernetes/OpenShift-spec memory limit string (e.g. "512Mi")
+      memoryLimit:       # Kubernetes/OpenShift-spec memory limit string (e.g. "512Mi")
+      memoryRequest:     # Kubernetes/OpenShift-spec memory request string (e.g. "256Mi")
+      cpuLimit:          # Kubernetes/OpenShift-spec CPU limit string (e.g. "500m")
```

Comment:
Consider referencing K8s documentation here https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu